dbt/include/redshift/macros/metadata/relation_last_modified.sql

```diff
@@ -0,0 +1,29 @@
+{% macro redshift__get_relation_last_modified(information_schema, relations) -%}
+
+    {%- call statement('last_modified', fetch_result=True) -%}
+        select
+            ns.nspname as "schema",
+            c.relname as identifier,
+            max(qd.start_time) as last_modified,
```

Comment:
Is there a reason to use `end_time` instead of `start_time`?

Reply:
I suppose because the start_time better reflects when the data being inserted was actually generated? (i.e. if an insert takes 2 hours the end_time would make the data look 2 hours "fresher" than it is)

Reply:
in most cases they probably won't be that different tho

Reply:
Yeah, this was my rationale as well. I'll keep it as is then.